<!-- For bag or simulation: no serial, just open two topic drivers -->
<launch>
  <node pkg="nmea_navsat_driver" type="nmea_topic_driver" name="gps1_topic" output="screen"/>
  <node pkg="nmea_navsat_driver" type="nmea_topic_driver" name="gps2_topic" output="screen">
    <remap from="/nmea_sentence"  to="/nmea_sentence_2"/>
    <remap from="/fix"            to="/fix_2"/>
    <remap from="/vel"            to="/vel_2"/>
    <remap from="/time_reference" to="/time_reference_2"/>
  </node>
</launch>
